perl_modules/EBI/FGPT/Converter/GEO/SOFTtoMAGETAB.pm

```diff
@@ -849,7 +849,7 @@ sub write_magetab
 {
 	my ($self) = @_;
 
-	my $idf = File::Spec->catfile( $self->get_target_dir, $self->get_acc . ".idf.txt" );
+	my $idf = File::Spec->catfile( $self->get_target_dir, $self->get_acc . ".merged.idf.txt" );
```

Comment:
Are we sure that this is not used for any other use where the `merged` addition would break things? If this is a new module, I would rather have the module receive the path to the idf file (and hence the merged or not merged can be defined upstream) instead of figuring out the idf path from obscure config files.

Reply:
I have tested these changes, and it works. This module is exclusively used for converting GEO soft files to MAGETAB. This is an old module.

Reply:
If this is only used by GEO importer, I would prefer that you move it there. If you want to keep it here, please add a passing option to the method while keeping the default. Simply adding `merged` here is a no no.

Reply:
We have optional argument based renaming IDF now.